kaiax/builder/impl/getter.go

```diff
@@ -17,14 +17,41 @@
 package impl
 
 import (
+	"bytes"
+
 	"github.com/kaiachain/kaia/blockchain/types"
 	"github.com/kaiachain/kaia/common"
+	"github.com/kaiachain/kaia/crypto"
 	"github.com/kaiachain/kaia/kaiax/builder"
 )
 
+// buildDependencyIndices builds a dependency indices of txs.
+// Two txs with the same sender has an edge.
+// Two txs in the same bundle has an edge.
+// TxGenerator is not connected to any other txs.
```

Comment:
> // TxGenerator is not connected to any other txs.

Please update this comment

Reply:
It was removed in [d03e039](https://github.com/kaiachain/kaia/pull/272/commits/d03e0396a6f92b22715eac6dfa9b36f6fee5910e).